src/pages/components/pagination/usage.mdx

```diff
@@ -54,55 +59,377 @@ control for navigating to the next or previous page.
-move forward or backward through pages of data. Provide an
-[inline select](/components/select/usage) in which users can choose the page
-they wish to navigate to.
+In the pagination nav variant, an ellipsis button appears between pages to
+indicate that there are numerous pages to navigate to within the ellipsis button
+menu. We recommend never placing the ellipsis button at the beginning or end of
+a series of pages in the pagination component.
 
 <Row>
 <Column colLg={8}>
 
-![Example of pagination controls on data table](images/pagination-usage-1.png)
+![Placement of the pagination nav variant](images/pagination-usage-overflow-pagination-nav.png)
 
 </Column>
 </Row>
 
+<Caption>
+  Example of the ellipsis button for overflow content in the pagination nav
+  variant
+</Caption>
+
+### Further guidance
+
+For further content guidance, see Carbon's
+[content guidelines](https://carbondesignsystem.com/guidelines/content/overview/).
+
+## Pagination
+
+The pagination variant is typically a bar attached to the bottom of a data table
+to paginate its data.
+
+<Row>
+<Column colLg={12}>
+
+![Example of the pagination variant in a UI](images/pagination-usage-pagination-in-context.png)
+
+</Column>
+</Row>
+
+<Caption>Example of the pagination variant in a UI</Caption>
+
+### States
+
+The pagination variant has the select component and ghost icon button component
+nested within it and inherits their component states. For more information on
+these states, refer to the
+[select](https://carbondesignsystem.com/components/select/style/) and
+[ghost icon button](https://carbondesignsystem.com/components/button/style/)
+style tab guidance.
+
+### Interactions
+
+#### Mouse
+
+- Clicking on the select component will open the menu and reveal options.
+- To close the menu, choose a different option or click outside the menu area.
+- Clicking on a previous or next button will navigate you to the previous or
+  next page.
+
+#### Keyboard
+
+- On focus, the select component menu is opened with pressing `Space`or with
+  `Up` or `Down` arrows which also cycle through the values.
+- Pressing `Space` or`Enter` selects a value and closes the menu.
+- The menu can be closed by pressing `Esc`.
+- The previous and next ghost icon buttons are activated by pressing `Space` or
+  `Enter`.
+
+### Responsive behavior
+
+The pagination variant retains all its content and nested components at every
+breakpoint except for the small breakpoint. The select components have been
+removed at the small breakpoint, but information on the total number of items,
+items being displayed, and previous and next buttons remain.
+
+<Row>
+<Column colLg={12}>
+
+![Pagination variant responsive behavior in all breakpoints versus the small breakpoint](images/pagination-usage-responsiveness-pagination.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Pagination variant responsive behavior in all breakpoints versus the small
+  breakpoint
+</Caption>
+
+### Clickable areas
```

Comment:
Are you missing content here or does it just not need any for this section?

Reply:
Yes, definetly missing content in there and did not copy it from Figma. Will update!